bzt/engine.py

```diff
@@ -948,6 +963,8 @@ class Scenario(UserDict, object):
     def __init__(self, engine, scenario=None):
         super(Scenario, self).__init__()
         self.engine = engine
+        if isinstance(scenario, string_types):
```

Comment:
This is bad. It does not play well with concept of passing existing scenario and modifications to it. This case should not exist.